panel/src/config/i18n.js

```diff
@@ -1,9 +1,29 @@
 
 import str from "../helpers/string.js";
 
+/**
+ * TranslationString prop type
+ */
+export class TranslationString {
+  constructor(key) {
+    this.key = key;
+  }
+  toString(app) {
+    return app.$t(this.key);
+  }
+}
```

Comment:
For maximum compatibility (e.g. `.length`):

```suggestion
export class TranslationString extends String {
  toString(app) {
    return app.$t(this.valueOf());
  }
}
```